ncache-vm-cluster/azuredeploy.parameters.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "value": "dataCache,sessionCache"
+    },
+    "cacheTopology": {
+      "value": "Partitioned,PartitionedOfReplica"
+    },
+    "cacheSize": {
+      "value": "256,512"
+    },
+    "numberOfVMs": {
+      "value": 2
+    },
+    "firstName": {
+      "value": "John"
+    },
+    "lastName": {
+      "value": "Smith"
+    },
+    "company": {
+      "value": "Alachisoft"
+    },
+    "emailAddress": {
+      "value": "john_smith@alachisoft.com"
+    },
+    "environmentName": {
+      "value": "NotSpecified"
+    },
+    "numberOfClients": {
+      "value": 4
+    },
+    "licenseKey": {
+      "value": "NotSpecified"
+    },
+    "virtualMachineNamePrefix": {
+      "value": "NCacheVM"
+    },
+    "virtualMachineSize": {
+      "value": "Standard_DS2_v2"
+    },
+    "adminUsername": {
+      "value": "ncadmin"
+    },
+    "adminPassword": {
+      "value": "12345Alachisoft"
+    },
+    "addressPrefix": {
+      "value": "10.0.0.0/16"
+    },
+    "subnetName": {
+      "value": "Subnet1"
+    },
+    "subnetPrefix": {
+      "value": "10.0.0.0/24"
+    },
+    "virtualNetworkName": {
+      "value": "MyVNET1"
+    },
+    "NCacheClusterCreationScriptFileUri": {
+      "value": "https://yourstorageaccount.blob.core.windows.net/tourcontainer/NCacheConfiguration.ps1"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts